modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase

```diff
@@ -81,12 +81,41 @@ definition {
 	}
 
 	@description = "This is a stubbed description"
-	@ignore = "true"
 	@priority = "5"
 	test CanClearInput {
+		NavItem.gotoStructures();
 
-		// TODO LRQA-70033	Refactor of Renderer Web Content Field
+		WebContentStructures.addCP(structureName = "WC Structure Title");
+
+		DEBuilder.addField(
+			fieldLabel = "Web Content",
+			fieldName = "Web Content");
+
+		WebContentStructures.saveCP();
+
+		NavItem.gotoWebContent();
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Title");
+
+		PortletEntry.inputTitle(title = "Web Content Title");
 
+		DERenderer.inputDataInWebContentField(
+			fieldLabel = "Web Content",
+			webContentWebContent = "Web Content Title 1");
+
+		Button.clickPublish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		DERenderer.clearImageField(fieldLabel = "Web Content");
```

Comment:
@guivilarim, I think it's time to rename this macro, after all you're cleaning a Web Content Field and not an Image Field.